corehq/apps/consumer_user/signals.py

```diff
@@ -1,32 +1,39 @@
-from corehq.apps.consumer_user.models import ConsumerUserInvitation
-from corehq.apps.consumer_user.models import ConsumerUserCaseRelationship
+from corehq.apps.consumer_user.models import (
+    ConsumerUserCaseRelationship,
+    ConsumerUserInvitation,
+)
 from corehq.form_processor.models import CommCareCaseSQL
 from corehq.form_processor.signals import sql_case_post_save
-from .const import CONSUMER_INVITATION_CASE_TYPE
-from .tasks import create_new_invitation
-
 
-def make_invitation_inactive(invitation, case):
-    invitation.make_inactive()
-    ConsumerUserCaseRelationship.objects.filter(case_id=case.case_id,
-                                                domain=case.domain).delete()
+from .const import (
+    CONSUMER_INVITATION_ACCEPTED,
+    CONSUMER_INVITATION_CASE_TYPE,
+    CONSUMER_INVITATION_SENT,
+    CONSUMER_INVITATION_STATUS,
+)
+from .tasks import create_new_invitation
 
 
 def send_email_case_changed_receiver(sender, case, **kwargs):
     if case.type != CONSUMER_INVITATION_CASE_TYPE:
         return
     email = case.get_case_property('email')
+    status = case.get_case_property(CONSUMER_INVITATION_STATUS)
     invitation = ConsumerUserInvitation.objects.filter(case_id=case.case_id,
                                                        domain=case.domain,
                                                        active=True).last()
+    is_status_sent_or_accepted = status == CONSUMER_INVITATION_SENT or status == CONSUMER_INVITATION_ACCEPTED
```

Comment:
can you add a test for this so it is clearer what is happening?

Reply:
Added the test to resend mail with existing logic